### ISSUE TITLE
Assorted distribution: propagation and combined distribution

### ISSUE THREAD

**Issue by adrik29** [SpeciesFileGroup/taxonworks#3053]
### Feature or enhancement

Asserted distribution records are a bit cumbersome in the current state:

1) First of all, the species distribution can be entered at the "Asserted Distribution" windows in the "**_Browse nomenclature and classification_**" module, while the supraspecific distribution is found in: "**_Task: Taxon name distribution_**".  

2) Those distribution assertions should propagate both taxonomically and geographically. For example, I have two species of genus Aus --  Aus bus in Texas and Aus cus in California.
So I should be able to know that [the species of] genus Aus occur in the USA.

3) Combined distribution of a supra-specific taxon is important. In the above example, the combined distribution of [the species of] genus Aus is California + Texas.
Currently in the "**_Task: Taxon name distribution_**" module there is only a 20-item window for a given taxon. It should be possible to choose ALL the species of a genus or a family for example. How do I know otherwise if a given family is endemic to the Palearctic?
![image](https://user-images.githubusercontent.com/46284943/180605880-c957b5f3-e768-44f0-898d-97f3aba07c6b.png)
![image](https://user-images.githubusercontent.com/46284943/180605895-e55b5f40-230d-4f0a-abed-e13416e17a23.png)

### Location

probably "Browse OTUs" but also "Browse nomenclature and classification"

### Your role

curator of project

**Comment by proceps** [comment]
We have discussed this for public pages, this should be also implemented in Browse OTU interface.

**Comment by adrik29** [comment]
@proceps -- Thanks, Dmitry. I'm glad to hear that we will be finally able to properly use the geographic units. This part seems neglected in TW for me. In the current state there's is not much we can do with the distribution we already entered. This Texas + California not resulting in USA is vexing.

The geo-units, even as they are now (without further Brazilian subdivision) would be very useful, so one can answer those questions:
1) What species of family Xidae occur in Europe?
2) Whats is the combined distribution of all 30 species of genus Cus? Note that I mentioned 30 especially because there is this 20 item window which I cannot reframe or enlarge.
Cheers, Adrik

**Comment by adrik29** [comment]
As TW stands right now, is it possible for example to make a checklist of the species occurring in a given country? If so, how?

**Comment by proceps** [comment]
You can make a list of species only in a single geographic area (it will not combine subdivisions). The task is 'Basic endemism'

**Comment by adrik29** [comment]
Thanks, Dmitry -- but it doesn't propagate?
I mean, if a species occur in Texas and California it will not appear in the USA list?!?!?

**Comment by adrik29** [comment]
@proceps -- I have tested this tool. It words very well as it is, that means, it is a promising start.
I noticed that it propagates the taxonomy but NOT the areas.

**Comment by proceps** [comment]
No, only one geographic area at the moment. Matt was working on new OTU filter interface, it should provide propagating functionality.

**Comment by adrik29** [comment]
Thanks for telling me about this tool. This seems to be heading for something fantastic!
